language/move-command-line-common/src/character_sets.rs

```diff
@@ -18,7 +18,7 @@ pub fn is_permitted_printable_char(c: char) -> bool {
 /// The only permitted newline character is \n. All others are invalid.
 pub fn is_permitted_newline_char(c: char) -> bool {
     let x = c as u32;
-    x == 0x0A
+    x == 0x0A || x == 0x0D
```

Comment:
Thanks for the PR! And sorry you are having issues, but we won't be able to change this right away.
This restriction was made in response to a potential security vulnerability where comments in Move code commented out the following line, without the programmer knowing. 

We had discussed relaxing this late last year, but did not make any firm plans. We could file an issue and will need to discuss this further